microsoft/botframework-components
# refactor: Move Telephony tests to tests\unit

### Purpose
Update telephony tests to align with rest if component tests 

### Changes
- Move Telephony tests to tests\unit
- Update the SDK version to 4.14.1
- Remove unused reference in .gitignore
- 
### Tests

- Ensure Microsoft.Bot.Components.Tests.sln builds successfully
- Ensure all tests run + telephony tests run successfully

#minor
